README.md

```diff
@@ -528,6 +528,38 @@ You can also set `"service" : "lambda"` to use a lambda instance.
 
 You can get the `accessKey` and `secretKey` from your AWS user, under programmatic access.
 
+#### Cloudflare
```

Comment:
```suggestion
#### Cloudflare AI Workers
```

Reply:
Solved in [f482e00](https://github.com/huggingface/chat-ui/pull/972/commits/f482e000513f0da87dd60ac3e4e5d334d5f16ec7), changed to Cloudflare Workers AI since that's [the official name](https://developers.cloudflare.com/workers-ai/)